app/src/main/java/com/github/libretube/enums/ContentFilter.kt

```diff
@@ -0,0 +1,26 @@
+package com.github.libretube.enums
+
+import com.github.libretube.constants.PreferenceKeys.SELECTED_FEED_FILTERS
+import com.github.libretube.helpers.PreferenceHelper
+
+private val enabledFiltersSet get() = PreferenceHelper
+    .getString(SELECTED_FEED_FILTERS, "1,2,3")
+    .split(',')
+    .toMutableSet()
```

Comment:
```suggestion
private val defaultFilters = arrayOf(ContentFilter.VIDEOS, ContentFilter.SHORTS, ContentFilter.LIVESTREAMS)

private val enabledFiltersSet get() = PreferenceHelper
   .getString(SELECTED_FEED_FILTERS, defaultFilters.joinToString(","))
    .split(',')
    .toMutableSet()
```

Additionally, please also move these two values into the companion object of the enum class `ContentFilter`, just since we're doing so too in other places to keep the things that belong together at one class.